.buildkite/scripts/test_e2e.sh

```diff
@@ -8,13 +8,20 @@
 ############################################################
 
 
-# Temporary test base directory.
-TEST_BASE_DIR=$(realpath ${TEST_BASE_DIR:-$(mktemp -d --tmpdir oasis-runtime-e2e-XXXXXXXXXX)})
 
 # Defaults.
 WORKDIR=$(pwd)
 TEST_FILTER=""
 
+# We need a directory in the workdir so that Buildkite can fetch artifacts.
+if [[ "${BUILDKITE:-""}" != "" ]]; then
```

Comment:
```suggestion
if [[ ! -z ${BUILDKITE} ]]; then
```
would also work, I think